listenbrainz/webserver/views/playlist_api.py

```diff
@@ -152,6 +153,113 @@ def serialize_jspf(playlist: Playlist):
     return {"playlist": pl}
 
 
+def serialize_xspf(playlist: Playlist):
+    """
+        Given a playlist, return a stringified element tree
```

Comment:
It would be good to add a comment here saying that/how the XML is buggy and disabled for the time being.